traffic_portal/app/src/common/modules/table/servers/TableServersController.js

```diff
@@ -548,6 +551,18 @@ var TableServersController = function(servers, $scope, $state, $uibModal, $windo
 		}
 	};
 
+	$scope.onQuickSearchChanged = function() {
+		$scope.gridOptions.api.setQuickFilter(document.getElementById('quickSearch').value);
```

Comment:
You're using `ng-model`, so you don't need to do `document.getElementById`, just use the model value.

Reply:
oh duh. i added ng-model later...